main.tf

```diff
@@ -54,6 +54,8 @@ resource "azurerm_linux_web_app" "main" {
     "DOCKER_REGISTRY_SERVER_URL"          = "https://index.docker.io"
     "FLASK_APP"                           = "todo_app/app"
     "FLASK_ENV"                           = var.env
+    "LOG_LEVEL"                           = "DEBUG"

```

Comment:
This might well be something we want to be able to configure - it is something I expect teams to have different opinions on, but maybe we want Debugging on for dev but Info and higher for prod or similar. Switching this to a variable, even if we then just set a default value for most scenarios, might make sense here